project.clj

```diff
@@ -18,6 +18,10 @@
                  [byte-streams "0.2.1"]
                  [camel-snake-kebab "0.3.2"]
 
+                 [clj-http "2.1.0"]
+                 [clj-http-fake "1.0.2"]
+                 [clj-time "0.11.0"]
+
```

Comment:
Aleph (and manifold, the library it builds on), are the libs we (esp @derwolfe and I) currently use most. It has a number of awesome features; notably asynchronous IO support in a way that is compatible with many different other libs, explicit backpressure, &c. I'm not sure it's worth switching; I'll read through the rest of the PR and think about it.